ansible/idr-elasticsearch.yml

```diff
@@ -70,6 +81,24 @@
       group: root
       mode: 0755
 
+
+
+  - name: Add elastic nodes to instances_nodes
+    set_fact:
+      instances_nodes: "{{instances_nodes + [( {'name' : 'searchengine_elasticsearch_node'+item, 'dns': ['searchengine_elasticsearch_node'+item,'localhost'],'ip': '127.0.0.1'})] }}"
+
+    with_sequence: start=1 count={{ elasticsearch_no_nodes }}
+
+
+  - name: Add elastic nodes to instances
+    set_fact:
+      instances: "{{instances | combine ( {'instances' : instances_nodes}) }}"
+
+
+  - name: Display the instances_nodes
+    debug: var=instances
```

Comment:
Is this debug statement necessary as the instances variable is serialized into a YAML file?

Reply:
I was using it for testing, I will remove it.